xls/ir/package.h

```diff
@@ -59,6 +59,7 @@ class Package {
            owned_function_types_.end();
   }
 
+  absl::StatusOr<Type*> GetType(const Type* type);
```

Comment:
Just to be very clear what this does, how about calling this MapTypeFromOtherPackage or something like that. And add a method comment.

Reply:
Done.